src/setups/extrude_geometry.jl

```diff
@@ -0,0 +1,212 @@
+"""
+    ExtrudeGeometry(geometry; particle_spacing, direction, n_extrude=1,
+                    velocity=zeros(length(direction)),
+                    mass=nothing, density=nothing, pressure=0.0)
+Extrude either a line, a plane or a shape along a specific direction.
+
+# Arguments
+- `geometry`:           Either points defining a 3D plane (2D line)
+                        or particle coordinates defining a specific shape.
+
+# Keywords
+- `particle_spacing`:   Spacing between the particles.
+- `direction`:          Vector defining the extrusion direction.
+- `n_extrude=1`         Number of `geometry` layers in extrude direction.
```

Comment:
```suggestion
- `n_extrude=1`       Specifies the number of layers of geometry particles created in the direction of extrusion. The default value is 1.
```

Reply:
That was my point. There shouldn't be a default value for this.

Reply:
no, I'll change it to `n_extrude::Integer`